ui/helpers/constants/routes.ts

```diff
@@ -282,5 +287,6 @@ export {
   ONBOARDING_EXPERIMENTAL_AREA,
   DESKTOP_ERROR_ROUTE,
   DESKTOP_PAIRING_ROUTE,
+  ADD_SNAP_ACCOUNT_ROUTE,
```

Comment:
This line is also inside the `build-flask` fence

Reply:
moved back into snaps in 479f66fbe539993ca94c06c0519910f29cfe8ff2